src/components/Maps/SearchBar.tsx

```diff
@@ -116,16 +116,17 @@ const SearchBar = ({
             disableEnforceFocus
           >
             <List classes={{ root: styles.sidebarList }}>
-              {!loading && suggestions.map((suggestion) => (
-                <ListItem
-                  key={suggestion.description}
-                  classes={{ root: styles.sidebarOptions }}
-                >
-                  <div {...getSuggestionItemProps(suggestion)}>
-                    <Typography>{suggestion.description}</Typography>
-                  </div>
-                </ListItem>
-              ))}
+              {!loading &&
+                suggestions.map((suggestion) => (
+                  <ListItem
+                    key={suggestion.description}
+                    classes={{ root: styles.sidebarOptions }}
+                  >
+                    <div {...getSuggestionItemProps(suggestion)}>
+                      <Typography>{suggestion.description}</Typography>
+                    </div>
+                  </ListItem>
+                ))}
```

Comment:
let's revert this, i'd rather not have random formatting changes check in especially if the other changes within the commit are unrelated

Reply:
hmm i think the linter (`npm run lint`) did this though, and if I revert I think it won't pass the `dev-lint` test

Reply:
somebody might've forgotten to lint their files before merging, which is why this change is being made. I think a similar thing happened with Ricky's recent PR. That being said, should I still revert?

Reply:
I made sure I linted SearchBar before merging it in...this should not be happening.